packages/web-components/src/components/va-accordion/va-accordion-item.tsx

```diff
@@ -153,7 +154,12 @@ export class VaAccordionItem {
           <div class={accordionItemClass}>
             <Header/>
             <slot name="headline" onSlotchange={() => this.populateStateValues()} />
-            <div id="content" class="usa-accordion__content usa-prose" hidden={!open}>
+            <div
+              id="content"
+              class="usa-accordion__content usa-prose"
+              hidden={!open}
+              part="accordion-content"
```

Comment:
Sorry @Mottie one more question. Why does this `part` need to be added? This isn't in the v1 variation. It's basically just the div around a slot, right? Couldn't the slot content be styled without it?

Reply:
There is also a `part='accordion-subheader'` in the original version that I don't see yet in the USWDS version. Would it be worthwhile to add that now while we're here?

https://github.com/department-of-veterans-affairs/component-library/blob/7f372260364bad0bb49573dc2546ad4b0567d993/packages/web-components/src/components/va-accordion/va-accordion-item.tsx#L197

Reply:
The `part="accordion-content"` is needed to style the borders around the content - you can see them in the screenshot, and updated as [part of this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/28841).

I don't know anything about the `part="accordion-subheader"` - I couldn't find any styling/restyling for it in vets-website. I don't think it's needed. Previously, it might have been needed to add the icon to the HTML accordion? I'm not sure.

Reply:
Yes, you're right. I don't see any style for that in vets-website either. 🤔  It was added as part of this issue for the MHV app:

- https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/1709

We can do a follow-up on this if they still need it in the USWDS variation. Thanks for checking.